charts/app/templates/appealReplyServiceApi.yaml

```diff
@@ -88,6 +88,23 @@ spec:
                   name: {{ $queueConfig.amqp.connection.existingUsernameSecret | quote }}
             - name: HORIZON_HAS_PUBLISHER_QUEUE
               value: {{ $queueConfig.amqp.receiver.source | quote }}
+            - name: DOCUMENTS_SERVICE_API_URL
+              value: "http://{{ include "app.fullname" . }}:{{ .Values.documentServiceApi.service.port }}"
+            - name: APPEALS_SERVICE_API_URL
+              value: "http://{{ include "app.fullname" . }}:{{ .Values.appealsServiceApi.service.port }}"
+            - name: SRV_NOTIFY_BASE_URL
+              value: { { .Values.appealsServiceApi.config.notify.baseUrl | quote } }
+            - name: SRV_NOTIFY_SERVICE_ID
+              value: { { .Values.appealsServiceApi.config.notify.serviceId | quote } }
+            - name: SRV_NOTIFY_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: value
+                  name: { { .Values.appealsServiceApi.config.notify.secretName } }
+            - name: SRV_NOTIFY_TEMPLATE_ID
+              value: { { .Values.appealsServiceApi.config.notify.templateId | quote } }
+            - name: SRV_NOTIFY_APPEAL_REPLY_SUBMISSION_CONFIRMATION
+              value: { { .Values.appealsServiceApi.config.notify.templates.appealReplyNotificationEmailToLpa | quote } }
```

Comment:
I believe this may be invalid syntax

No spaces between `{ {` and `} }`

though I may be wrong